src/libs/ReportUtils.ts

```diff
@@ -326,7 +326,7 @@ type OptionData = {
     text: string;
     alternateText?: string | null;
     allReportErrors?: Errors | null;
-    brickRoadIndicator?: typeof CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR | '' | null;
+    brickRoadIndicator?: typeof CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR | typeof CONST.BRICK_ROAD_INDICATOR_STATUS.INFO | '' | null;
```

Comment:
Wouldn't something like this work?
```suggestion
    brickRoadIndicator?: ValueOf<typeof CONST.BRICK_ROAD_INDICATOR_STATUS> | '' | null;
```
I found it in: `/src/libs/UserUtils.ts`